datahub-web-react/src/app/domainV2/CreateDomainModal.tsx

```diff
@@ -119,8 +119,22 @@ export default function CreateDomainModal({ onClose, onCreate }: Props) {
     return (
         <Modal
             title="Create New Domain"
-            visible
+            open
             onCancel={onClose}
+            buttons={[
+                {
+                    text: 'Cancel',
+                    variant: 'text',
+                    onClick: onClose,
+                },
+                {
+                    text: 'Save',
+                    id: 'createDomainButton',
+                    buttonDataTestId: 'create-domain-button',
+                    onClick: onCreateDomain,
+                    disabled: !createButtonEnabled,
+                },
+            ]}
             footer={
```

Comment:
do we need to remove the `footer` here? it looks like maybe we just don't use that in our modal component?